### ISSUE TITLE
Feedback*

### ISSUE THREAD

**Issue by samdeane** [schwa/MastodonAgain#3]
(* or things I'm thinking I might try to change in a fork, for my own benefit)

The thing I'm after more than anything is a minimalist interface.

Here are some tweaks I'm thinking of making:

- option to hide the the user's handle: seeing their name should be enough most of the time; if I hover over it (or long-press on iOS), then the handle can be shown
- option to hide the status actions unless you're hovering over (or have selected?) a toot
- show reblogs as (**Original Poster** (via **Reblogger**) - I think the original author's name should be the most important one

**Comment by samdeane** [comment]
(obviously feel free to completely ignore this - I just figured I might as well tell you what I was doing in case any of it was useful or prompted some redesign)

**Comment by samdeane** [comment]
![image](https://user-images.githubusercontent.com/206306/203307768-ffe738f4-8b65-4ced-bb81-2347836d2c0a.png)

**Comment by schwa** [comment]
Have you seen the toggle in the window bar for the "single line" mode? Very (too) minimalist too.

**Comment by samdeane** [comment]
Yeah, that was a little too extreme for me!

This is where I'm at currently:

<img width="577" alt="image" src="https://user-images.githubusercontent.com/206306/203360969-f2e19071-e586-4971-808a-ad2db227fadb.png">

**Comment by schwa** [comment]
If you put up a PR I'll take a look. It doesn't necessary have to be an option. I've not focused too much on the UX yet.

**Comment by samdeane** [comment]
I've put up #7, but it's all very subjective so don't worry if my changes don't appeal :)

It's an alternative mode but basically LargeStatusRow so could be swapped in for that.

**Comment by schwa** [comment]
Can we break the work up into smaller chunks? I'm leery of wholesale changes to the UI here.

**Comment by schwa** [comment]
(Changed issue title.)

**Comment by samdeane** [comment]
yeah, that's what I was doing before you suggested putting up a PR - pulling out self-contained changed from here and submitting them as individual PRs. I'll carry on in that vein.
